libs/@guardian/core-web-vitals/README.md

```diff
@@ -0,0 +1,140 @@
+# `@guardian/core-web-vitals`
+
+[![npm (scoped)](https://img.shields.io/npm/v/@guardian/core-web-vitals)](https://www.npmjs.com/package/@guardian/core-web-vitals)
+[![ES version](https://badgen.net/badge/ES/2020/cyan)](https://tc39.es/ecma262/2020/)
+[![npm type definitions](https://img.shields.io/npm/types/@guardian/core-web-vitals)](https://www.typescriptlang.org/)
+[![Coverage Status](https://coveralls.io/repos/github/guardian/libs/badge.svg)](https://coveralls.io/github/guardian/libs)
```

Comment:
This coverage is out of date and not applicable to this repo, should we remove it entirely?

Reply:
good point - will remove from libs too